libclc/cmake/modules/AddLibclc.cmake

```diff
@@ -28,7 +28,7 @@ function(compile_to_bc)
   get_filename_component( FILE_EXT ${ARG_INPUT} EXT )
   if( NOT ${FILE_EXT} STREQUAL ".ll" )
     # Pass '-c' when not running the preprocessor
-    set( PP_OPTS -c )
+    set( PP_OPTS -c -include opencl-c-base.h )
```

Comment:
Maybe should be using -finclude-default-header or -fdeclare-opencl-builtin instead? 

I'm assuming this picks up the clang builtin copy, not the duplicated name one in libclc?